src/main/java/friday/task/Task.java

```diff
@@ -0,0 +1,116 @@
+package friday.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a general task in the Friday application.
+ * This is the base class for different types of tasks such as Todo, Deadline, and Event.
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
```

Comment:
All your booleans are named to sound like booleans, great job 👍

Reply:
Thx!